packages/experiments/src/components/Stack/Stack.test.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import { mergeStyles } from 'office-ui-fabric-react';
+
+import { Stack } from './Stack';
+
+const sampleClass = mergeStyles({
+  background: 'red'
+});
+
+describe('Stack', () => {
+  it('renders vertical Stack correctly', () => {
+    const component = renderer.create(
+      <Stack>
+        <div>Item 1</div>
+        <div>Item 2</div>
+      </Stack>
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders horizontal Stack correctly', () => {
+    const component = renderer.create(
+      <Stack horizontal>
+        <div>Item 1</div>
+        <div>Item 2</div>
+      </Stack>
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders Stack with vertical and horizontal alignment correctly', () => {
+    const component = renderer.create(
+      <Stack horizontalAlignment="start" verticalAlignment="end">
+        <div>Item 1</div>
+        <div>Item 2</div>
+      </Stack>
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders Stack with vertical and horizontal fill correctly', () => {
+    const component = renderer.create(
+      <Stack fillHorizontal fillVertical>
+        <div>Item 1</div>
+        <div>Item 2</div>
+      </Stack>
+    );
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('renders Stack with grow correctly', () => {
+    const component = renderer.create(
```

Comment:
Snapshots are a good first step but all of these are best captured as visual regression tests.

Reply:
I was planning on adding vr tests in a separate PR (possibly when Stack is getting moved to oufr). I could also add them in this PR - thoughts?

Reply:
It is up to you :)